pkg/controller/workspace/env/common_env_vars.go

```diff
@@ -10,50 +10,49 @@
 //   Red Hat, Inc. - initial API and implementation
 //
 
-package component
+package env
 
 import (
+	"github.com/che-incubator/che-workspace-operator/pkg/config"
 	corev1 "k8s.io/api/core/v1"
-
-	. "github.com/che-incubator/che-workspace-operator/pkg/controller/workspace/model"
 )
 
-func commonEnvironmentVariables(wkspCtx WorkspaceContext) []corev1.EnvVar {
+func CommonEnvironmentVariables(workspaceName, workspaceId, namespace string) []corev1.EnvVar {
 	return []corev1.EnvVar{
 		{
 			Name: "CHE_MACHINE_TOKEN",
 		},
 		{
 			Name:  "CHE_PROJECTS_ROOT",
-			Value: DefaultProjectsSourcesRoot,
+			Value: config.DefaultProjectsSourcesRoot,
 		},
 		{
 			Name:  "CHE_API",
-			Value: wkspCtx.CheApiExternal,
+			Value: config.DefaultApiEndpoint,
 		},
 		{
 			Name:  "CHE_API_INTERNAL",
-			Value: DefaultApiEndpoint,
+			Value: config.DefaultApiEndpoint,
 		},
 		{
 			Name:  "CHE_API_EXTERNAL",
-			Value: wkspCtx.CheApiExternal,
+			Value: config.DefaultApiEndpoint,
 		},
 		{
 			Name:  "CHE_WORKSPACE_NAME",
-			Value: wkspCtx.WorkspaceName,
+			Value: workspaceName,
 		},
 		{
 			Name:  "CHE_WORKSPACE_ID",
-			Value: wkspCtx.WorkspaceId,
+			Value: workspaceId,
 		},
 		{
 			Name:  "CHE_AUTH_ENABLED",
-			Value: AuthEnabled,
+			Value: config.AuthEnabled,
 		},
 		{
 			Name:  "CHE_WORKSPACE_NAMESPACE",
-			Value: wkspCtx.Namespace,
+			Value: namespace,
 		},
```

Comment:
As a quick solution for CloudShell with OpenShiftOAuth: 
```suggestion
		},
		{
			Name:  "USE_BEARER_TOKEN",
			Value: config.ControllerCfg.GetWebhooksEnabled(),
		},
```
As a bit better solution I wonder if OpenShift OAuth Routing could contribute env var to all containers via PodAdditions... But then we have another question that SA with exec rights is not needed anymore, then who should contribute it?
I'm OK with trying to solve this issue in scope of a dedicated issue.